CRM/Core/BAO/CustomField.php

```diff
@@ -109,11 +109,8 @@ public static function create($params) {
   }
 
   /**
-   * Create/ update several fields at once in a mysql efficient way.
```

Comment:
Even thought it is deprecated we should still have a sentence about the function here

Reply:
Ok updated.